js/bkex.js

```diff
@@ -1654,6 +1654,59 @@ module.exports = class bkex extends Exchange {
         return result;
     }
 
+    async fetchFundingRateHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
@Dan-krm I'm getting this "param error" here
![image](https://user-images.githubusercontent.com/43336371/207010121-e615cfc1-9014-446c-885c-80d8d2f8ed72.png)